app/src/main/java/com/hamdan/forzenbook/data/repository/CreateAccountRepositoryImpl.kt

```diff
@@ -1,7 +1,10 @@
 package com.hamdan.forzenbook.data.repository
 
-import android.util.Log
 import com.hamdan.forzenbook.data.network.CreateAccountService
+import java.sql.Date
+
+private val SUCCESS = 200..299
```

Comment:
Companion object

Reply:
removed the range according to a different comment.